packages/office-ui-fabric-react/src/components/Persona/Persona.styles.ts

```diff
@@ -224,7 +224,6 @@ export const getStyles = (props: IPersonaStyleProps): IPersonaStyles => {
 
       showSecondaryText && {
         height: showSecondaryTextDefaultHeight,
-        lineHeight: showSecondaryTextDefaultHeight,
```

Comment:
removing `lineHeight` will affect the vertical alignment of the text

Reply:
I don't repro the issue in our example page https://developer.microsoft.com/en-us/fabric#/controls/web/persona, which i believe uses default theme. in the case you have the repro, what are the `font-size`, `height` and `lineHeight`?

Reply:
hey @xugao, thanks for pointing out that it isn't reproducible on the demo page. that led me to do some more digging and while i think this change wouldn't hurt (particularly for the comment left by @philipzloh), i'll close this PR as the OWA-specific problem can be fixed within OWA code.